requirements.txt

```diff
@@ -1,8 +1,9 @@
 -i https://pypi.org/simple
 PyJWT<2  # Bad overlap in azure dependencies
-httplib2
+httplib2>=0.15.0
 google-api-python-client
-google-auth
+google-auth<2.0.0.dev0
+google-api-core<1.26.0.dev0
```

Comment:
Are these changes voluntary? I remember unpinning httplib2 since the bug I had pinned it for was fixed.

Reply:
I was getting failures in the automated tests run by github until these version restrictions. Not sure on the actual why though.

Reply:
OK, let's leave them as is for now, I'll circle back on dependencies anyways since I need to fix some e2e tests that are failing.